docs/more-info/react-native.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: "React Native"
+description: "Our package is a wrapper for our iOS SDK."
+---
+
+<Info>
+  As a prerequisite [visit the apple health docs](/more-info/apple-health#initial-setup) and take a
+  look at the initial setup.
+</Info>
+
+<Warning>
+  This widget is compatible with Apple. If you would like to see how to implement on android [check
+  the example below.](/more-info/react-native#usage)
+</Warning>
+
+## Installation
+
+```sh
+npm install metriport-react-native
+```
+
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd ios
+pod install
+```
+
+## Usage
+
+To use our SDK go to the root of your project and add:
+
+```jsx
+import * as React from "react";
+
+import { StyleSheet, Platform } from "react-native";
+import { MetriportWidget } from "@metriport/react-native-sdk";
+import { WebView } from "react-native-webview";
+
+export default function App() {
+  if (Platform.OS === "ios") {
+    return (
+      <MetriportWidget
+        token="CONNECT_SESSION_TOKEN"
+        clientApiKey="CLIENT_API_KEY"
+        sandbox={false}
+        style={styles.box}
+      />
+    );
+  }
+
+  return <WebView source={{ uri: "https://connect.metriport.com/?token=demo" }} />;
+}
```

Comment:
General comment:

While we're at it, should we obfuscate this implementation from the end user?

ie `MetriportWidget` can be platform agnostic, and will handle all the URL params + logic internally.

So devs won't have to use raw URLs for Android, and will have a single interface/component for both platforms.